src/ui-kit/form-controls/autocomplete-multiselect/autocomplete-cache.spec.ts

```diff
@@ -118,3 +118,19 @@ describe('Sam Autocomplete Cache Class', () => {
     expect(cache.lastAdded).toEqual(moreValues);
   });
 });
+
+describe('Autocomplete Cache arraysEqual method', () => {
+  const initialArray = [{ key: 1 }, { key: 2 }, { key: 3 }, { key: 4 }];
```

Comment:
Add some more tests to cover more than just array:

https://github.com/lodash/lodash/blob/d5ef31929a1262abbc75b8dadc0b6ae6e9558b5f/test/isEqual.js